src/clj/ataru/virkailija/virkailija_routes.clj

```diff
@@ -119,10 +119,19 @@
                            :path-params [application-id :- Long]
                            :summary "Return application details needed for application review, including events and review data"
                            :return {:application ataru-schema/Application
-                                    :events      [ataru-schema/Event]}
+                                    :events      [ataru-schema/Event]
+                                    :review      s/Any}
                            (trying (fn []
                                      {:application (application-store/get-application application-id)
-                                      :events      (application-store/get-application-events application-id)})))
+                                      :events      (application-store/get-application-events application-id)
+                                      :review      (application-store/get-application-review application-id)})))
+
+                   (api/PUT "/review" []
```

Comment:
I think the request path should be  `/:applicationId/review` since the application under review already is a known resource (instead of just hiding the application ID under the review object).